hapi-utils/src/main/java/com/hederahashgraph/fee/FeeBuilder.java

```diff
@@ -58,10 +58,9 @@
   public static final int THRESHOLD_STORAGE_TIME_SEC = 90000;
   public static final int DEFAULT_RBS_NETWORK = 0;
   public static final int FEE_DIVISOR_FACTOR = 1000;
-  public static final int SIGNATURE_SIZE = 64;
   public static final int HRS_DIVISOR = 3600;
   public static final int BASIC_ENTITY_ID_SIZE = (3 * LONG_SIZE);
-  public static final long BASIC_RICH_INSTANT_SIZE = (1L * LONG_SIZE) + INT_SIZE;
+  public static final long BASIC_RICH_INSTANT_SIZE = ((long) LONG_SIZE) + INT_SIZE;
```

Comment:
wouldn't this be the same to multiply with 1L instead of casting ?

Reply:
would be same but removed unnecessary arithmetic operation.

Reply:
Got it !! But, it was added since we did not want many casts while cleaning up sonar cloud minor bugs.

Reply:
Ahh I see. will revert this back